modules/client-configuration/pages/clients-opensuse.adoc

```diff
@@ -0,0 +1,133 @@
+[[clients-opensuse]]
+= Registering openSUSE Clients
+
+This section contains information about registering Salt clients running {ubuntu} operating systems.
+
+{productname} supports opensuse Leap 15 and 15.1 Clients using Salt.
+// Traditional clients are not supported.
+
+// [IMPORTANT]
+// ====
+// {suse} does not endorse or support {productname}.
+// ====
+
+Bootstrapping is supported for starting openSUSE clients and performing initial state runs such as setting repositories and performing profile updates.
+
+
+
+== Prepare to Register
+
+Some preparation is required before you can register {ubuntu} clients to the {productname} Server.
+
+////
+ifeval::[{suma-content} == true]
+
+// If we need this for openSUSE, cf. clients-ubuntu.adoc
+.Procedure: Adding client tools channels
+
+etc.
+
+////
+
+ifeval::[{uyuni-content} == true]
+.Procedure: Adding the openSUSE Channels
+
+. At the command prompt on the {productname} Server, as root, install the [systemitem]``spacewalk-utils`` package:
++
+----
+zypper in spacewalk-utils
+----
+. Add the openSUSE channels.
+For openSUSE Leap 15:
++
+----
+spacewalk-common-channels \
+opensuse_leap15_0 \
+opensuse_leap15_0-non-oss \
+opensuse_leap15_0-non-oss-updates \
+opensuse_leap15_0-updates \
+opensuse_leap15_0-uyuni-client
+----
++
+For openSUSE Leap 15.1:
++
+----
+spacewalk-common-channels \
+opensuse_leap15_1 \
+opensuse_leap15_1-non-oss \
+opensuse_leap15_1-non-oss-updates \
+opensuse_leap15_1-updates \
+opensuse_leap15_1-uyuni-client
+----
+
+. Synchronize the new custom channels.
+
+[IMPORTANT]
+====
+You need all the new channels fully synchronized before bootstrapping any openSUSE client.
+openSUSE channels can be very large.
+Synchronization can sometimes take several hours.
+====
+
+To synchronize, navigate to menu:Software[Manage > Channels], and for each channel you created, click it and:
+
+. Navigate to the [guimenu]``Repositories``.
+. Navigate to the [guimenu]``Sync``and click btn:[Sync Now] to begin synchronization immediately.
+
+You can also setup a scheduled syncronization from this screen.
+
+endif::[]
+
+== Monitor Synchronization Progress
+
+There are two ways to check if a channel has finished synchronizing:
+
+// ifeval::[{suma-content} == true]
```

Comment:
If I am not wrong, openSUSE Leap 15.1 is now at SCC, and that's what should be used for SUSE Manager.